helm/provisioner/templates/configmap.yaml

```diff
@@ -33,20 +33,21 @@ data:
   storageClassMap: |
     {{- range $classConfig := .Values.classes }}
     {{ $classConfig.name }}:
-       hostDir: {{ $classConfig.hostDir }}
-       mountDir: {{ if $classConfig.mountDir }} {{- $classConfig.mountDir -}} {{ else }} {{- $classConfig.hostDir -}} {{ end }}
-       {{- if $classConfig.blockCleanerCommand }}
-       blockCleanerCommand:
-       {{- range $val := $classConfig.blockCleanerCommand }}
-         - "{{ $val -}}"{{- end}}
-       {{- end }}
-       {{- if $classConfig.volumeMode }}
-       volumeMode: {{ $classConfig.volumeMode }}
-       {{- end }}
-       {{- if $classConfig.fsType }}
-       fsType: {{ $classConfig.fsType }}
-       {{- end }}
-       {{- if $classConfig.namePattern }}
-       namePattern: "{{ $classConfig.namePattern }}"
-       {{- end }}
+      hostDir:  {{ $classConfig.hostDir }}
```

Comment:
s/hostDir:  {{/hostDir: {{/

remove one space between `:` and `{{`

Reply:
Updated